ckanext/dcatapchharvest/profiles.py

```diff
@@ -503,15 +538,29 @@ def parse_dataset(self, dataset_dict, dataset_ref):  # noqa
                     ('download_url', DCAT.downloadURL),
                     ('url', DCAT.accessURL),
                     ('coverage', DCT.coverage),
-                    ('license', DCT.license),
             ):
                 value = self._object_value(distribution, predicate)
                 if value:
                     resource_dict[key] = value
 
-            # Rights
-            resource_dict['rights'] = self._rights(distribution, DCT.rights)
-
+            #  Rights & License
+            rights = self._rights(distribution, DCT.rights)
+            license = self._license(distribution, DCT.license)
+            
+            if rights is None and license is not None:
+                resource_dict['license'] = license.get('url')
+                resource_dict['rights'] = license.get('label')
+                if ('cc' in license.get('label')):
+                    resource_dict['license'] = license.get('label')
+                    resource_dict['rights'] = rights.get('label')
+                
+            if license is None and rights is not None:   
+                resource_dict['license'] = rights.get('url')
+                resource_dict['rights'] = rights.get('label')
+                if ('cc' in rights.get('label')):
+                    resource_dict['license'] = license.get('label')
+                    resource_dict['rights'] = rights.get('label')
```

Comment:
For both `license` and `rights`, we want to be able to do the following:
- display the license/rights name on opendata.swiss (a string)
- export the field as a `LicenseDocument` or `RightsStatement` with a URI

I think it's cleaner to save them both in the same format (string or URI) and use the same method to display/export them, rather than keeping the `rights` value as a string and `license` as a URI. I would suggest saving both as the URI - we will then need to get the license/rights name from the URI for displaying them on the CKAN website or in the API.